.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
     - 3.4
     - 3.5
     - 3.6
+    - 3.7
```

Comment:
The special 'dist' and 'sudo' values are missing here so 3.7 didn't build.

Reply:
D'oh!

Reply:
All fixed now!